charts/erzwingungshaft/templates/cronjob.yaml

```
@@ -0,0 +1,47 @@
+  schedule: {{ $module.cronjob.schedule | quote }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: erzwingungshaft
+            image: "{{ $module.image.registry }}/{{ $module.image.repository }}:{{ $module.image.tag }}"
+            imagePullPolicy: {{ $module.image.pullPolicy }}
+            envFrom:
+            - secretRef:
+                name: {{ $module.name }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)